app/controllers/cases_controller.rb

```diff
@@ -241,10 +241,10 @@ def correspondence_types_for_current_user
   end
 
   def add_sar_ir_to_permitted_types_if_sars_allowed(types)
-    types.delete(CorrespondenceType.sar_internal_review)
```

Comment:
@willmcb , this function don't need any more, you can remove this part entirely.  The idea is that if you choose to add the SAR IR into team's list when creating a new by default + data migration for existing team,  this part becomes redundant and also potentially cause inconsistent if there is change for case type permission . 

I tried your PR without this func, it works good ,  but it may affect the automated tests -> you may need to add this case type into the SAR team explicitly, give a try,  if too many cases are affected, you may keep this part, sorry for being picky 😳

Reply:
@ymao2 thanks for your comment - I was wondering about this but if you look at the way the types are collected from `current_user` you can see that it calls `menu_visible` which does not include the SAR IR correspondence type so I think that we need to specifically add it in in this case. It disappears from the "Create case" page if not.

Reply:
Hi @willmcb , I see, forgot the menu visible thing.